participants.php

```diff
@@ -199,14 +199,22 @@
 if ($bulkoperations) {
     echo '<br /><div class="buttons"><div class="form-inline">';
     echo '<input type="submit" name="submit" value="'.get_string('bulkchangedate', 'local_recompletion').'"/>';
+    echo '<input type="submit" name="reset_completion" value="'.get_string('bulkresetallcompletion', 'local_recompletion').'"/>';
+
     echo '<input type="hidden" name="id" value="' . $course->id . '" />';
     echo '</div></div>';
 }
 
 echo '</form>';
 
+
```

Comment:
please avoid adding extra spacing in areas that you haven't changed - can you please revert the spacing changes (including the ones below)